scripts/bash_aliases.sh

```diff
@@ -14,9 +14,8 @@ alias nav="cd $CATKIN_DIR/src/NaviGator"
 alias rnav="ros_connect -n ${HOSTNAMES[1]}"
 alias sshnav="ssh navigator@${HOSTNAMES[1]} -Y"
 
-# Missions
-alias navm="rosrun navigator_missions run_mission"
-alias navc="rosrun navigator_missions move_command"
+# Tasks
+alias nmove="runtask Move"
```

Comment:
why not navc/navm?

Reply:
felt like being explicit about aliases is nice. New people would probably forget navc cause it doesn't have to do with move

Reply:
ah okay, make sense